com.tracktion.Waveform11.yml

```diff
@@ -0,0 +1,128 @@
+id: com.tracktion.Waveform11
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+tags:
+  - proprietary
+command: waveform11
+finish-args:
+  - --device=all
+  - --env=LV2_PATH=$HOME/.lv2:/app/extensions/Plugins/lv2
+  - --env=LXVST_PATH=/app/extensions/Plugins/lxvst
+  - --env=VST3_PATH=/app/extensions/Plugins/vst3
+  - --env=LADSPA_PATH=/app/extensions/Plugins/ladspa
+  - --filesystem=xdg-config
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-download
+  - --filesystem=xdg-music
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-run/pipewire-0
+  - --filesystem=xdg-videos
```

Comment:
sadly limit the filesystem can possibly cause some plugins to not work.
It will also prohibit installing VST3 in .vst3

Reply:
What would you suggest here? `home`?

Reply:
that's usually the case. Not ideal though, even more since no one can see the source code. I'm just warning of the pitfalls here, and what you set could be a starting point though.

Reply:
I'll keep an eye on bug reports, I prefer relaxing constrained filesystem permissions than giving up at start